tests/test_cli/test_utils/test_package_utils.py

```diff
@@ -157,9 +157,14 @@ def test_update_aea_version_range() -> None:
     """Test `update_aea_version_range`"""
 
     package_config = AgentConfig("name", "author", aea_version="0.1.0")
-    update_aea_version_range(package_configuration=package_config)
+    assert not package_config.aea_version_specifiers.contains(
+        str(get_current_aea_version())
+    )
 
-    assert str(get_current_aea_version()) in package_config.aea_version
+    update_aea_version_range(package_configuration=package_config)
+    assert package_config.aea_version_specifiers.contains(
+        str(get_current_aea_version())
+    )
```

Comment:
Instead of performing a strict check we perform a range check here. This check is failing because the current version of open-aea is `1.27.0.post1` and `update_aea_version_range` creates the version range as `>=1.27.0, <2.0.0`. We were checking if the exact version version (`1.27.0.post1`) was in the version range string or not and that's why the test failed. Now we're performing a check in the range.

Issue for the same https://github.com/valory-xyz/open-aea/issues/523